parm/config/gefs/config.ufs

```diff
@@ -98,13 +98,13 @@ case "${fv3_res}" in
         export DELTIM=600
         export layout_x=2
         export layout_y=2
-        export layout_x_gfs=2
-        export layout_y_gfs=2
+        export layout_x_gfs=4
+        export layout_y_gfs=4
         export nthreads_fv3=1
         export nthreads_fv3_gfs=1
         export nthreads_ufs=1
         export nthreads_ufs_gfs=1
-        export xr_cnvcld=".false."  # Do not pass conv. clouds to Xu-Randall cloud fraction
+        export xr_cnvcld=".true."  # Pass conv. clouds to Xu-Randall cloud fraction
```

Comment:
@NeilBarton-NOAA we could perhaps just remove this xr_cnvcld variable from the workflow to avoid any confusion. It is default .true. in the code, and when we wanted it to be false for C96 only it made sense to keep it here. But now when we moved it back to true, and instead changed the scaling in the code - it is probably cleaner to just remove.

Reply:
If this is a general change and not just to shoehorn C96 SFS in until the separate run is available, it should be made for C48 as well.

Reply:
added for C48